docs/docs/gatsby-on-linux.md

```diff
@@ -64,10 +62,11 @@ Or to install all at the same time and approve `(y)` all installs:
 sudo apt update && sudo apt -y upgrade && sudo apt install build-essential && sudo apt install git && sudo apt install libpng-dev
 ```
 
-<!-- links -->
+### Additional links and resources
 
-[windows store]: https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6
-[n]: https://github.com/tj/n
-[n-install]: https://github.com/mklement0/n-install
-[nvm]: https://github.com/creationix/nvm
-[bash startup]: https://github.com/Microsoft/WSL/issues/776#issuecomment-266112578
+- [Super detailed guide to making VSCode work with ESL from VSCode's docs website](https://code.visualstudio.com/docs/remote/wsl)
```

Comment:
Small typo

```suggestion
- [Super detailed guide to making VSCode work with WSL from VSCode's docs website](https://code.visualstudio.com/docs/remote/wsl)
```

Reply:
Yes! I will add more detailed instructions while adding instructions to the part 0 page! I could split them by package manager too, like mentioned there.